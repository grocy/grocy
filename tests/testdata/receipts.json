{
	"data": [{
			"address": {
				"address1": "8867 161st Ave NE",
				"address2": "Redmond, WA 98052"
			},
			"items": [{
					"itemIdentifier": "0004000052550",
					"baseUpc": "0004000052550",
					"department": "4",
					"unitOfMeasure": "EA",
					"detail": {
						"upc": "0004000052550",
						"customerFacingSize": "9.7 oz",
						"description": "Twix Minis Size Chocolate Cookie Bars",
						"familyTreeCommodityCode": "202",
						"familyTreeDepartmentCode": "04",
						"familyTreeSubCommodityCode": "3227",
						"mainImage": "https://www.kroger.com/product/images/small/front/0004000052550",
						"soldInStore": true,
						"categories": [{
								"category": "Candy",
								"categoryCode": "77"
							}
						]
					},
					"quantity": 1,
					"totalSavings": 1.79,
					"extendedPrice": 4.29,
					"pricePaid": 2.50,
					"priceModifiers": [{
							"type": "1",
							"amount": 1.79,
							"action": "Subtract",
							"promotionId": "10000019468",
							"couponType": "73",
							"reportingCode": "0"
						}
					],
					"unitPrice": 4.29,
					"itemType": "NORMAL",
					"entryMethod": "KEYED_ITEM_CODE",
					"imageUrl": "https://www.kroger.com/product/images/small/front/0004000052550",
					"detailUrl": "/p/item/0004000052550",
					"sellableUpc": "0004000052550",
					"fuel": false,
					"giftCard": false,
					"managerOverrideIgnored": false,
					"containerDeposit": false,
					"pharmacy": false,
					"managerOverride": false,
					"weighted": false
				}, {
					"itemIdentifier": "0007222000516",
					"baseUpc": "0007222000516",
					"department": "75",
					"unitOfMeasure": "EA",
					"detail": {
						"upc": "0007222000516",
						"customerFacingSize": "6 ct / 13 oz",
						"description": "Franz Extra Crisp English Muffins",
						"familyTreeCommodityCode": "050",
						"familyTreeDepartmentCode": "75",
						"familyTreeSubCommodityCode": "2407",
						"mainImage": "https://www.kroger.com/product/images/small/front/0007222000516",
						"soldInStore": true,
						"categories": [{
								"category": "Breakfast",
								"categoryCode": "10"
							}
						]
					},
					"quantity": 3,
					"totalSavings": 6.9,
					"extendedPrice": 12.87,
					"pricePaid": 5.97,
					"priceModifiers": [{
							"type": "1",
							"amount": 2.3,
							"action": "Subtract",
							"promotionId": "10000013849",
							"couponType": "73",
							"reportingCode": "0"
						}, {
							"type": "1",
							"amount": 2.3,
							"action": "Subtract",
							"promotionId": "10000013849",
							"couponType": "73",
							"reportingCode": "0"
						}, {
							"type": "1",
							"amount": 2.3,
							"action": "Subtract",
							"promotionId": "10000013849",
							"couponType": "73",
							"reportingCode": "0"
						}
					],
					"unitPrice": 4.29,
					"itemType": "NORMAL",
					"entryMethod": "KEYED_ITEM_CODE",
					"imageUrl": "https://www.kroger.com/product/images/small/front/0007222000516",
					"detailUrl": "/p/item/0007222000516",
					"sellableUpc": "0007222000516",
					"fuel": false,
					"giftCard": false,
					"managerOverrideIgnored": false,
					"containerDeposit": false,
					"pharmacy": false,
					"managerOverride": false,
					"weighted": false
				},
				{
					"itemIdentifier": "0000000004648",
					"baseUpc": "0000000004648",
					"department": "57",
					"unitOfMeasure": "LBR",
					"detail": {
						"upc": "0000000004648",
						"customerFacingSize": "1 lb",
						"description": "Mushrooms - Crimini - Bulk",
						"familyTreeCommodityCode": "613",
						"familyTreeDepartmentCode": "57",
						"familyTreeSubCommodityCode": "8067",
						"mainImage": "https://www.kroger.com/product/images/small/front/0000000004648",
						"soldInStore": true,
						"categories": [{
								"category": "Produce",
								"categoryCode": "36"
							}
						]
					},
					"quantity": 0.84,
					"totalSavings": 0,
					"extendedPrice": 3.35,
					"pricePaid": 3.35,
					"priceModifiers": [],
					"unitPrice": 3.99,
					"itemType": "NORMAL",
					"entryMethod": "KEYED_ITEM_CODE",
					"imageUrl": "https://www.kroger.com/product/images/small/front/0000000004648",
					"detailUrl": "/p/item/0000000004648",
					"sellableUpc": "0000000004648",
					"containerDeposit": false,
					"managerOverride": false,
					"weighted": true,
					"giftCard": false,
					"pharmacy": false,
					"fuel": false,
					"managerOverrideIgnored": false
				}
			],
			"transactionTime": "2020-03-21T09:14:00"
		}
	]
}
