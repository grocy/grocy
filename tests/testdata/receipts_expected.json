[{
    "name": "Twix Minis Size Chocolate Cookie Bars",
    "location_id": 4,
    "qu_id_purchase": 3,
    "qu_id_stock": 3,
    "barcode": "0040000525509",
    "quantity": 1,
    "transaction_date": "2020-03-21"
}, {
    "name": "Franz Extra Crisp English Muffins",
    "location_id": 3,
    "qu_id_purchase": 3,
    "qu_id_stock": 3,
    "barcode": "0072220005165",
    "quantity": 3,
    "transaction_date": "2020-03-21"
}, {
    "name": "Mushrooms - Crimini - Bulk",
    "location_id": 3,
    "qu_id_purchase": 2,
    "qu_id_stock": 2,
    "qu_factor_purchase_to_stock": 1,
    "barcode": "0000000046480",
    "default_best_before_days": 21,
    "quantity": 0.84,
    "transaction_date": "2020-03-21",
    "price_paid": 3.988095238095238,
    "picture_url": "https:\/\/www.kroger.com\/product\/images\/small\/front\/0000000004648"
}
]